deployments/container/Makefile

```diff
@@ -28,7 +28,9 @@ IMAGE = $(IMAGE_NAME):$(IMAGE_TAG)
 
 ubuntu22.04: BASE_IMAGE = docker.io/ubuntu:22.04
 
-# Use a generic build target to build the relevant images
+ubi9: BASE_IMAGE = registry.access.redhat.com/ubi9/ubi-micro:9.4
```

Comment:
This is not specific to using podman? Could we move this to a different PR. I would prefer to not have to maintain this in this repo and could look at allowing a user to set `BASE_IMAGE` instead to target other distributions.

The concern is keeping the tags up to date, for example.